demos/ec2/README.md

```diff
@@ -0,0 +1,43 @@
+# configure Amazon EC2 Plguin
+
+https://wiki.jenkins.io/display/JENKINS/Amazon+EC2+Plugin
+
+## sample configuration
+
+```yaml
+jenkins:
+  clouds:
+  - amazonEC2:
+      cloudName: "docker-agents"
+      credentialsId: "jenkins-aws"
+      privateKey: "-----BEGIN RSA PRIVATE KEY-----\n\
```

Comment:
Probably best not to encourage putting private keys in the config file?
```suggestion
      privateKey: "${EC2_PRIVATE_KEY}
```

Reply:
Good point. It's updated in the code now.